big_tests/tests/offline_SUITE.erl

```diff
@@ -41,6 +43,23 @@ suite() ->
 
 init_per_suite(C) -> escalus:init_per_suite(C).
 end_per_suite(C) -> escalus_fresh:clean(), escalus:end_per_suite(C).
+
+init_per_group(with_groupchat, C) ->
+    Modules = [{mod_offline, [{store_groupchat_messages, true}]},
```

Comment:
I think it's worth to specify `mod_offline`'s backend as well. Otherwise mnesia will be used when the module is restarted for the group.